plugins/analysis-icu/src/main/java/org/elasticsearch/index/analysis/IcuNormalizerTokenFilterFactory.java

```diff
@@ -35,14 +38,15 @@
  * <p>The {@code unicodeSetFilter} attribute can be used to provide the UniCodeSet for filtering.</p>
  */
 public class IcuNormalizerTokenFilterFactory extends AbstractTokenFilterFactory implements MultiTermAwareComponent {
-
+    private final static DeprecationLogger deprecationLogger =
```

Comment:
@romseygeek this line is causing checkstyle violations
```
> Task :plugins:analysis-icu:checkstyleMain FAILED
[ant:checkstyle] [ERROR] /home/alpar/work/elastic/elasticsearch/plugins/analysis-icu/src/main/java/org/elasticsearch/index/analysis/IcuNormalizerTokenFilterFactory.java:41:19: 'static' modifier out of order with the JLS suggestions. [ModifierOrder]
```
I'm pushing a fix to master now. FYI

Reply:
Master was merged in a few days ago and the checkstyle rules last changed 4 weeks ago, yet the PR build passed.

Reply:
I opened #35207 to fix these